NewEvaluation.css

```diff
@@ -190,9 +204,22 @@ input[type=submit]{
 	float: right;
 	box-shadow: 5px 5px 5px #888888;
 }
-.spacing{
-	padding-top: 25vh;
-}
-.littleSpacing{
-	padding-top: 10vh;
+
+
+
+
+
+/*-----------------   Footer -----------------*/
+
+
+.BottomFooter{
+	bottom:0;
+    position:fixed;
+    z-index:150;
+    _position:absolute;
+    _top:expression(eval(document.documentElement.scrollTop+
```

Comment:
And did it actually work the way you intended it to work? I'm really curious, because a) I've never played around with CSS expressions myself and b) I still find the extra `_` before the property name rather dubious.